demo.cfg

```diff
@@ -1,2 +1,30 @@
 [buildout]
 extends = buildout.cfg
+# Uncomment to just test ini build
+extends = ini-templates/ini_buildout.cfg
+ini_template = ini-templates/production-template.ini
+ini_name = demo.ini
+# app
+accession_factory = encoded.server_defaults.test_accession
```

Comment:
If variables like these are going to just be static, then wouldn't it be better to put them in the ini-template? It will be easier to move to a different templating engine later if they stay in `ini` files, rather `cfg` which is more buildout specific. Same goes for the other cfg/ini files.

Reply:
I think was resolved in current update.